src/controller/java/BUILD.gn

```diff
@@ -65,7 +65,7 @@ android_library("java") {
   ]
 
   sources = [
-    "gen/ChipClusters.java",
+    "gen/chip/devicecontroller/ChipClusters.java",
```

Comment:
You'll need to update `java/templates/templates.json` to have ZAP generate `ChipClusters.java` into this new directory.

https://github.com/project-chip/connectedhomeip/blob/df9aa588edf6a7a603cc5a9e07e3e71156fd88c6/src/controller/java/templates/templates.json#L37

Reply:
I changed this json file but when I am running ./scripts/tools/zap_regen_all.py, it got below errors, anything I missed?

LiHuideMacBook-Pro:connectedhomeip lihui$ ./scripts/tools/zap_regen_all.py
internal/modules/cjs/loader.js:968
  throw err;
  ^

Error: Cannot find module 'yargs'
Require stack:
- /Users/lihui/Code/matter/connectedhomeip/third_party/zap/repo/src-script/zap-generate.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:965:15)
    at Function.Module._load (internal/modules/cjs/loader.js:841:27)
    at Module.require (internal/modules/cjs/loader.js:1025:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (/Users/lihui/Code/matter/connectedhomeip/third_party/zap/repo/src-script/zap-generate.js:19:15)
    at Module._compile (internal/modules/cjs/loader.js:1137:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)
    at Module.load (internal/modules/cjs/loader.js:985:32)
    at Function.Module._load (internal/modules/cjs/loader.js:878:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/lihui/Code/matter/connectedhomeip/third_party/zap/repo/src-script/zap-generate.js'
  ]
}
Traceback (most recent call last):
  File "./scripts/tools/zap/generate.py", line 130, in <module>
    main()
  File "./scripts/tools/zap/generate.py", line 125, in main
    runGeneration(zap_file, zcl_file, templates_file, output_dir)
  File "./scripts/tools/zap/generate.py", line 83, in runGeneration
    subprocess.check_call(['node', './src-script/zap-generate.js', '-z', zcl_file, '-g', templates_file, '-i', zap_file, '-o', output_dir])
  File "/Users/lihui/Code/matter/connectedhomeip/.environment/cipd/python/lib/python3.8/subprocess.py", line 364, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['node', './src-script/zap-generate.js', '-z', '/Users/lihui/Code/matter/connectedhomeip/src/app/zap-templates/zcl/zcl.json', '-g', '/Users/lihui/Code/matter/connectedhomeip/src/app/zap-templates/app-templates.json', '-i', '/Users/lihui/Code/matter/connectedhomeip/examples/lighting-app/lighting-common/lighting-app.zap', '-o', '/Users/lihui/Code/matter/connectedhomeip/examples/lighting-app/lighting-common/gen/']' returned non-zero exit status 1.
Traceback (most recent call last):
  File "./scripts/tools/zap_regen_all.py", line 61, in <module>
    main()
  File "./scripts/tools/zap_regen_all.py", line 58, in main
    subprocess.check_call(['./scripts/tools/zap/generate.py'] + target)
  File "/Users/lihui/Code/matter/connectedhomeip/.environment/cipd/python/lib/python3.8/subprocess.py", line 364, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['./scripts/tools/zap/generate.py', 'examples/lighting-app/lighting-common/lighting-app.zap']' returned non-zero exit status 1.

Reply:
updated at ccb80f0d5cfc421356d03811eebd2e3df56a9d6b